README.md

```diff
@@ -34,11 +34,18 @@ Supported Cameras
 
 Installing
 --------
+**Prerequisite:**
```

Comment:
Can we just include `mogrify.exe` in the bin folder and dynamically get the path to it so user's don't have to do this extra step?

Reply:
I just check the license.txt of ImageMagick (mogrify) and it seems to be possible to include it to the focus-point distribution:
---
It allows you to:

  * freely download and use ImageMagick software, in whole or in part, for personal, company internal, or commercial purposes;
  * use ImageMagick software in packages or distributions that you create;
  * link against a library under a different license;
  * link code under a different license against a library under this license;
  * merge code into a work under a different license;
  * extend patent grants to any code using code under this license;
  * and extend patent protection.

It forbids you to:

  * redistribute any piece of ImageMagick-originated software without proper attribution;
  * use any marks owned by ImageMagick Studio LLC in any way that might state or imply that ImageMagick Studio LLC endorses your distribution;
  * use any marks owned by ImageMagick Studio LLC in any way that might state or imply that you created the ImageMagick software in question.

It requires you to:

  * include a copy of the license in any redistribution you may make that includes ImageMagick software;
  * provide clear attribution to ImageMagick Studio LLC for any distributions that include ImageMagick software.

It does not require you to:

  * include the source of the ImageMagick software itself, or of any modifications you may have made to it, in any redistribution you may assemble that includes it;
  * submit changes that you make to the software back to the ImageMagick Studio LLC (though such feedback is encouraged).

Reply:
I can try to do this tomorrow...

Reply:
Done. I've committed the following changes in ropma/windows-support:

Windows Support - add ImageMagick to plugin distribution
- add ImageMagick to bin folder
- InfoProvider: remove Options section (not needed anymore as mogrify is part of the plugin distribution)
- InfoProvider: add Acknowledgements section
- change license type of MogrifyUtils.lua
- update README.md

Test on a "clean" windows box was successful, thus the ImageMagick installation in the plugin bin folder is self contained.

Reply:
I will leave this PR up for a day or 2 incase there is anything you want to change and then will merge it in to the branch. I'll give a QA pass on Mac and then merge it over. Once I tell people it works on Windows, you're probably going to get a huge influx of people using it (and probably giving feedback on how it works). Thanks for your effort on this. Couldn't have done it without you!

Reply:
Also thanks to you for creating and maintaining this project!! I really enjoy to use the plugin.

Great if you can do a QA on Mac. I don't have one and only did some basic tests with the  overlay mode after refactoring.